src/gui/folderwizard.h

```diff
@@ -60,6 +60,16 @@ class FolderWizardLocalPath : public FormatWarningsWizardPage
     void cleanupPage() override;
 
     void setFolderMap(const Folder::Map &fm) { _folderMap = fm; }
+
+signals:
+    void styleChanged();
+
+public slots:
+    void slotStyleChanged();
+
+protected:
+    void changeEvent(QEvent *) override;
```

Comment:
why is this one public ?

Reply:
Not sure what you mean, do you mean why is `slotStyleChanged` public or why isn't `changeEvent` public?

Reply:
Alex clarified below, addressed this